models/quix/optimism/quix_v2_optimism_events.sql

```diff
@@ -0,0 +1,231 @@
+{{ config(
```

Comment:
@chuxinh did you forget to add a posthook or do you want to keep this spell out of the data explorer for now?

Reply:
yea that's correct. intentionally leaving this blank

Reply:
👍